src/ansible_creator/subcommands/add.py

```diff
@@ -256,6 +261,14 @@ def _perform_module_plugin_scaffold(
         resources = ("collection_project.plugins.sample_module",)
         self._perform_plugin_scaffold(resources, template_data, plugin_path)
 
+    def _perform_test_plugin_scaffold(
+        self,
+        template_data: TemplateData,
+        plugin_path: Path,
+    ) -> None:
+        resources = ("collection_project.plugins.test",)
```

Comment:
Is it possible to make this line as: `resources = (f"collection_project.plugins.{self._plugin_type}",)` ?